enterprise/migrations/versions/076_add_v1_tables.py

```diff
@@ -0,0 +1,274 @@
+"""Sync DB with Models
+
+Revision ID: 076
+Revises: 075
+Create Date: 2025-10-05 11:28:41.772294
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+from openhands.app_server.app_conversation.app_conversation_models import (
+    AppConversationStartTaskStatus,
+)
+from openhands.app_server.event_callback.event_callback_result_models import (
+    EventCallbackResultStatus,
+)
+from openhands.integrations.service_types import ProviderType
+from openhands.storage.data_models.conversation_metadata import ConversationTrigger
+
+# revision identifiers, used by Alembic.
+revision: str = '076'
+down_revision: Union[str, Sequence[str], None] = '075'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    """Upgrade schema."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        'v1_app_conversation_info',
```

Comment:
I'd say we drop the `v1_` prefix on these tables. It's easy enough to change in the future I suppose, but it's one of those things that gets left behind